README.md

```diff
@@ -198,14 +198,139 @@ Example:
 }
 ```
 
+### Filtering Content Items
+
+**Note:** Filtering content via `filterBy() | filterByContentType() | filterByParentId() | filterContentTypes()` is only supported when using [Content Delivery 2](#content-delivery-versions)
+
+Once you have [a schema set up for filtering](https://amplience.com/docs/development/contentdelivery/filterandsort.html), the content item must be published before it can be retrieved using this SDK.
+
+### Constructing a request
+
+The `filterBy() | filterByContentType() | filterByParentId() method will return a instance of the`FilterBy` class which has helper functions to construct a filterBy request.
+
+Calling `request` executes the request returning a `Promise` if no content is found an empty response object will be returned. If invalid options are provided it will reject with an error.
+
+```ts
+const client = new ContentClient({
+  hubName: 'myhub',
+});
+
+interface BlogPost {
```

Comment:
I like the example that shows the use of generics, but think we should start with a simpler JS example.

Reply:
fair play, i can do without